bindings/python/crocoddyl/multibody/frames.cpp

```diff
@@ -6,15 +6,17 @@
 // All rights reserved.
 ///////////////////////////////////////////////////////////////////////////////
 
+#include "crocoddyl/multibody/frames.hpp"
+
 #include <eigenpy/memory.hpp>
 #include <eigenpy/eigen-to-python.hpp>
 
+
+#include "pinocchio/bindings/python/utils/std-aligned-vector.hpp"
```

Comment:
Could you use `<>` simbol (as it is an external header)?

Reply:
sure